spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/SystemFunctionPushDownHelper.java

```diff
@@ -39,6 +56,22 @@ public static void createPartitionedTable(
     insertRecords(spark, tableName);
   }
 
+  public static void createPartitionedTable(
+      SparkSession spark, String tableName, String partitionCol, Map<String, String> props) {
+    if (props.isEmpty()) {
+      createPartitionedTable(spark, tableName, partitionCol);
+    } else {
+      String tblProps = Joiner.on("','").withKeyValueSeparator("'='").join(props);
```

Comment:
Iceberg does not needlessly abbreviate words, like "table" as "tbl". It would also be more descriptive to use `propsString` because the significant part is that the properties were converted to a string.

Reply:
Updated